### PR TITLE
DT-1136: 💄 Add in nunjucks configuration

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ jobs:
           key: gradle-cache-{{ checksum "manage-accounts-specs/build.gradle" }}
       - run:
           name: Run Integration Tests
-          command: ./gradlew test
+          command: ./gradlew check
```
Comment (petergphillips) [inline .circleci/config.yml:55]: from previous PR - `check` runs all the tests (not that in practise there is any difference)

File: backend/dateHelpers.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const moment = require('moment')
```
Comment (petergphillips) [inline backend/dateHelpers.js:1]: copied from PSH

File: backend/index.js [hunk 3]
```diff
@@ -79,8 +84,8 @@ app.use(noCache())
 app.use(bodyParser.urlencoded({ extended: false }))
 app.use(bodyParser.json())
 
-app.use('/bundle.js', express.static(path.join(__dirname, '../build/bundle.js')))
```
Comment (petergphillips) [inline backend/index.js:82]: moved to the static content file

File: backend/nunjucksSetup.js [hunk 4]
```diff
@@ -0,0 +1,146 @@
+const moment = require('moment')
+const nunjucks = require('nunjucks')
+const config = require('./config')
+const { getDate, getTime, pascalToString, capitalize, hyphenatedStringToCamel } = require('./utils')
```
Comment (petergphillips) [inline backend/nunjucksSetup.js:4]: copied from PSH

File: backend/setupSass.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+const express = require('express')
```
Comment (petergphillips) [inline backend/setupSass.js:1]: Copied from PSH

File: backend/setupStaticContent.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+const express = require('express')
```
Comment (petergphillips) [inline backend/setupStaticContent.js:1]: Copied from PSH

File: backend/utils.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
+const moment = require('moment')
```
Comment (petergphillips) [inline backend/utils.js:1]: Copied from PSH

File: backend/utils.test.js [hunk 8]
```diff
@@ -0,0 +1,54 @@
+import { capitalize, getDate, getTime } from './utils'
```
Comment (petergphillips) [inline backend/utils.test.js:1]: Copied from PSH

File: sass/application.scss [hunk 9]
```diff
@@ -0,0 +1,191 @@
+$govuk-page-width: 1170px;
```
Comment (petergphillips) [inline sass/application.scss:1]: Copied from PSH

**Review by JamieDraperUK — APPROVED** [review]
lgtm!

**Pull request merged**